src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -2133,8 +2128,8 @@ private Boolean fixSELinuxLabel(File key, String label) {
 
             // Keep track which "clues" indicate that SELinux currently is
             // a force that should be reckoned with, on this host now:
-            Boolean clue_proc = false;
-            Boolean clue_sysfs = false;
+            boolean clue_proc = false;
+            boolean clue_sysfs = false;
```

Comment:
This change makes me nervous because it requires investigation to confirm that it does not alter the serialization.  I don't have time to do that investigation as part of the review of the pull request.  If you're willing to remove this change from the pull request, I'm willing to merge the pull request.  If you feel strongly that this must be included in the pull request before it is merged, then this pull request will need to wait until I have time to assess the impact on serialization.
```suggestion
            Boolean clue_proc = false;
            Boolean clue_sysfs = false;
```

Reply:
Thanks for your review. I have reverted the change.